src/tiffimage_int.cpp

```diff
@@ -2042,89 +2041,98 @@ bool TiffHeaderBase::isImageTag(uint16_t /*tag*/, IfdId /*group*/, const Primary
   return false;
 }
 
+static bool isTiffImageTagLookup(uint16_t tag, IfdId group) {
+  if (group != IfdId::ifd0Id) {
+    return false;
+  } else {
```

Comment:
No need to use else after return.

Reply:
Right, by doing so we remove a level of indentation in the whole `switch` statement.